tests/DomTestCaseTest.php

```diff
@@ -1008,6 +1008,7 @@ public function testAssertXPathCountPresentTrue()
     /**
      * @covers            PHPUnit_Framework_DOMTestCase::assertXPathCount
      * @expectedException PHPUnit_Framework_AssertionFailedError
+     * @expectedExceptionMessage Failed asserting that false is true.
```

Comment:
I think we should work on the assertion messages so that it’s understandable by the message why the test failed.